app/src/main/java/com/github/libretube/services/AbstractPlayerService.kt

```diff
@@ -95,6 +95,7 @@ abstract class AbstractPlayerService : MediaLibraryService(), MediaLibrarySessio
             super.onPlaybackStateChanged(playbackState)
 
             if (playbackState == Player.STATE_READY) {
+                exoPlayer?.let { PlayerHelper.saveWatchPosition(it, videoId) }
```

Comment:
Rethinking it, shouldn't this be in `Player.STATE_IDLE`? `STATE_READY` is triggered when a new, other video starts playing.

Reply:
No, I just misread `STATE_READY` as `STATE_ENDED` :)

Reply:
Oh right, it's been called `STATE_ENDED`, not `STATE_ENDED` :)